api/src/main/scala/dcos/metronome/api/v1/controllers/ApplicationController.scala

```diff
@@ -19,6 +21,18 @@ class ApplicationController(metrics: Metrics) extends RestController {
 
   def ping = Action { Ok("pong") }
 
+  def info = Action {
+
+    val infoJson = Json.toJson(
```

Comment:
maybe we should introduce a case class for this and serialize it using some marshaller ... ?

Reply:
I don't see the value... less code is better code.